HTML5SDK/wwtlib/Layers/SpreadSheetLayer.cs

```diff
@@ -2031,6 +2030,108 @@ public int SizeColumn
                 }
             }
         }
+
+        // The following attributes control whether the point sizes should be normalized before
+        // being used. When NormalizeSize is true, the point sizes are scaled using
+        //
+        // new_size = (size - NormalizeSizeMin) / (NormalizeSizeMax - NormalizeSizeMin)
+        //
+        // The NormalizeSizeClip attribute can be used to determine whether the sizes should
+        // be clipped to the range [0:1]. At this time, normalization is only applied if
+        // PointScaleTypes is Linear or Power.
```

Comment:
I'm wondering if we can figure out something better for the clipping behavior.

First of all, regardless of whether the point size is measured in pixels or degrees, it's not obvious to me that a value of 1 is a natural maximum to clip to. If the final computed size can be interpreted as a number of pixels (caveat: I do not know if this is indeed the case), clipping to a range of 0-1 seems quite restrictive.

The other thing is that clipping behavior at 0 and 1 seems pretty asymmetrical. My guess would be that negative radii would render as their absolute values, i.e. a radius of -1 is the same as a radius of 1. This could be hard to spot as a user. Meanwhile, radii above 1 almost certainly have continuous behavior.

I think I'd suggest that rather than a boolean clip flag, there be a floating-point `ClipMax` attribute above which values are always clipped. It could default to +Inf. Radii below 0 would always be clipped to 0.

In such a model, it might make sense to rename the `NormalizeSizeMax` parameter to `NormalizeSizeUnity`, or something like that.

Reply:
The way I was using this currently was to use the min/max to get values in the range [0:1], then using the clip on that, and then use the ``scaleFactor`` setting to scale the points to whatever absolute size I want, so e.g. using ``scaleFactor`` of 10 would lead to clipped values in the range [0:10]. This does mean that the clip value only makes sense when the min/max are also specified, and requires ``scaleFactor`` to be specified.

Reply:
Oh, the `scaleFactor` also gets applied in this chain? Then yes, this makes more sense.

Reply:
Yes it gets applied, not in the methods I wrote, but in the next step.